src/views/AidDetail.vue

```diff
@@ -47,9 +47,10 @@
 </template>
 
 <script>
-    import Header from "@/components/Header";
+    import aidService from '../services/aidService'
```

Comment:
```suggestion
    import aidService from '@/services/aidService'
```

C'est vraiment du pinaillage, mais je trouve ça plus cohérent.